src/mapping/VolumeCellInterpolation.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <list>
+#include <string>
+#include <vector>
```

Comment:
We separate these by a blank line.

```suggestion
#include <vector>

```